### ISSUE TITLE
Which parts of Steam networks system contains vulnerabilities ?

### ISSUE THREAD

**Issue by Pordrack** [Facepunch/Facepunch.Steamworks#643]
Hello.
Recently, there has been quite the ruccus with the multiplayers aspects of the Steam multiplayer API, with like, Dark souls players getting hacked.
What exactly is it ? Is it the Steam Networking P2P system ? The SteamNetworkingSockets part ? Are these vulnerabilites presents in Facepunch implementation ?

**Comment by userKolecko** [comment]
Hi, basically the old steam P2P system reveals your ip. Steam networking sockets don't do that, so you should be fine using those.

**Comment by marcospgp** [comment]
Indeed, and this wiki page should be marked as deprecated: https://wiki.facepunch.com/steamworks/Sending_P2P_Messages
